python/tk_multi_publish2/dialog.py

```diff
@@ -40,7 +40,98 @@ class AppDialog(QtGui.QWidget):
     (DRAG_SCREEN, PUBLISH_SCREEN) = range(2)
 
     # details ui panes
-    (ITEM_DETAILS, TASK_DETAILS, PLEASE_SELECT_DETAILS) = range(3)
+    (ITEM_DETAILS, TASK_DETAILS, PLEASE_SELECT_DETAILS, MULTI_EDIT_NOT_SUPPORTED) = range(4)
+
+    # settings ui panes, those are both present on the TASK_DETAILS pane.
+    (BUILTIN_TASK_DETAILS, CUSTOM_TASK_DETAILS) = range(2)
+
+    class TaskSelection(object):
```

Comment:
I am not a big fan of classes within classes. Can we make this `_TaskSelection` at the bottom of the file or something?